source/nodejs/tests/ui-tests/src/testUtils.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Webdriver from "selenium-webdriver";
+
+export class TestUtils {
+    driver: Webdriver.WebDriver;
+
+    constructor(driver: Webdriver.WebDriver) {
+        this.driver = driver;
+    }
+
+    async goToTestCase(testCaseName: string): Promise<void> {
+        const elementLinkText: Webdriver.WebElement = await this.driver.findElement(Webdriver.By.id(testCaseName));
+        await elementLinkText.click();
+    }
+
+    async getActionWithTitle(actionTitle: string): Promise<Webdriver.WebElement> {
+        const button: Webdriver.WebElement = await this.driver.findElement(Webdriver.By.xpath("//*[@aria-label='" + actionTitle + "']"));
```

Comment:
might be a little clearer if you use string interpolation:

```suggestion
        const button: Webdriver.WebElement = await this.driver.findElement(Webdriver.By.xpath(`//*[@aria-label='${actionTitle}']`));
```